README.md

```diff
@@ -29,20 +29,43 @@ The Dapr docs are built using [Hugo](https://gohugo.io/) with the [Docsy](https:
 
 The [daprdocs](./daprdocs) directory contains the hugo project, markdown files, and theme configurations.
 
-## Pre-requisites
+## Setup with a devcontainer
 
-- [Hugo extended version](https://gohugo.io/getting-started/installing)
-- [Node.js](https://nodejs.org/en/)
+This repository comes with a [devcontainer](/.devcontainer/devcontainer.json) configuration that automatically installs all the required dependencies and VSCode extensions to build and run the docs.
+
+This devcontainer can be used to develop locally with VSCode or via GitHub Codespaces completely in the browser. Other IDEs that support [devcontainers](https://containers.dev/) can be used but won't have the extensions preconfigured and will likely have different performance characteristics.
+
+### Pre-requisites
 
-## Environment setup
+- [Docker Desktop](https://www.docker.com/products/docker-desktop)
+- [VSCode](https://code.visualstudio.com/download)
 
-1. Ensure pre-requisites are installed
-2. Clone this repository
+### Environment setup
+
+1. [Fork](https://github.com/dapr/docs/fork) and clone this repository.
+
+2. Open the forked repository in VS Code
```

Comment:
```suggestion
1. Open the forked repository in VS Code
```

This is a nit, but Markdown automatically numbers lists with the correct order - keeping all items as `1.` helps when we edit later, so we don't have to remember to renumber lists if we add or remove an item.